e2e_tests/tests/cli.test.js

```diff
@@ -86,6 +86,49 @@ describe('KYT CLI', () => {
     expect(shell.test('-d', 'build/public')).toBe(true);
   });
 
+  it('runs the build command and exits on SIGINT', () => {
+    const exec = new Promise((resolve) => {
+      let finishedBuild = false;
+      let sentKill = false;
+      const child = shell.exec('npm run build', (code) => {
+        resolve({ finishedBuild, code });
+      });
+      child.stdout.on('data', (data) => {
+        if (data.includes('✅ Done building')) {
+          finishedBuild = true;
+          resolve(finishedBuild);
```

Comment:
pretty minor, but maybe this would be more readable if you rejected the promise here, and then later asserted that the promise is not rejected?